src/components/ActivityLog/ActivityForm.tsx

```diff
@@ -0,0 +1,84 @@
+import { zodResolver } from '@hookform/resolvers/zod';
+import Activity from 'interface/Activity';
+import { SubmitHandler, useForm } from 'react-hook-form';
+import { z } from 'zod';
```

Comment:
since you're using only object, coerce and string I recommend you to only import the used one